.kitchen.yml

```diff
@@ -9,12 +9,13 @@ provisioner:
 platforms:
   - name: ubuntu-12.04
   - name: ubuntu-14.04
-  - name: centos-6.4
+  - name: centos-6.7
   - name: centos-7
   - name: debian
   - name: arch
     driver:
-      image: base/arch
+      image: base/archlinux
+      provision_command:
```

Comment:
I use an empty provisioning command to prevent a Chef installation on Arch Linux